5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
@mfojtik I agree it's hard to figure out whole story behind the atomic labels, OTOH I don't agree it's that cryptic. Users will copy anything if they a) don't have reason to understand what they are doing b) can easily find good documentation.

There were ideas about making labels more declarative, but that would be again very problematic regarding backwards compatibility and ease of use. Single docker run command for each step makes sense to me (at least from what I can imagine as alternatives). You can easily inspect image and review the command, you can override it if needed...

Install (in most cases) needs to write to host, thus needs to run as privileged container. No, you don't need to always get rid of all namespaces, but sometimes you do. The analogy for this is `yum install` where, if you package some malicious code f.e. in `%post` script, you are in the same situation.

`OPTx` arguments are mess, I agree, though I have no idea how to fix them,,